Modules/_hashopenssl.c

```diff
@@ -336,10 +320,14 @@ static PY_EVP_MD*
 py_digest_by_name(PyObject *module, const char *name, enum Py_hash_type py_ht)
 {
     PY_EVP_MD *digest = NULL;
+#if 1
```

Comment:
leftover from debugging?

Reply:
I left it for you so you can reproduce my perf check. It's gone now.